### PR TITLE
Feature/tasks frontend

### PR DESCRIPTION
**Pull request by FistMeNaruto**

### CODE REVIEW

File: src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/inc/_taskList.html.twig [hunk 1]
```diff
@@ -1,16 +1,48 @@
-<ul class="collection" id="full-task-list-container">
-    {% for task in course.tasks %}
-        <li class="row collection-item valign-wrapper">
-            <div class="col s1 center-align valign">
-                <h4 class="{% if task.points < 0 %}red-text{% else %}green-text{% endif %}">{{ task.points }}</h4>
+{#<ul class="collection" id="full-task-list-container">#}
```
Comment (JanisVepris) [inline src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/inc/_taskList.html.twig]: Dafak is this?

File: src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/inc/_taskList.html.twig [hunk 2]
```diff
@@ -1,16 +1,48 @@
...
+                <i class="material-icons right card-title grey-text">close</i>
+                <p>{{ task.description }}</p>
             </div>
-        </li>
-    {% endfor %}
-</ul>
+        </div>
+
+
+        {#<div class="card medium">#}
```
Comment (JanisVepris) [inline src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/inc/_taskList.html.twig]: Dafak dafak

File: src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/taskList.html.twig [hunk 3]
```diff
@@ -6,20 +6,27 @@
...
+            <div id="full-task-list-container" class="row">
+                {% include "@EdukodasTemplate/Task/inc/_taskList.html.twig" with {
                 course: course
-            } %}
+                } %}
+            </div>
         </div>
 
-        <div id="full-task-list-preloader" style="display: none;">
+
```
Comment (JanisVepris) [inline src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/taskList.html.twig]: Kam tiek daug tusciu eiluciu?

File: src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/inc/_taskList.html.twig (hunk 2 above)
Comment (FistMeNaruto) [reply src/Edukodas/Bundle/TemplateBundle/Resources/views/Task/inc/_taskList.html.twig]: VAJA ACIPRASAU BAISIAUSEI <3

**Pull request merged**